### ISSUE TITLE
Add global shortcuts for basic addon interaction

### ISSUE THREAD

**Issue by Intuos5** [hlorus/CAD_Sketcher#140]
Currently there is no alphanumerical hotkey for the Solvespace select tool. Although <kbd>Esc</kbd> and Right mouse click can be used to locate the tool, it is common for CAD software to invoke the move command from a dedicated shortcut.
I propose to add this to all tools in addition to right clicking.
![afbeelding](https://user-images.githubusercontent.com/63149655/170308156-e8445758-faf0-44f5-b091-2d0f96c3ea33.png)

I would personally use <kbd>W</kbd> for this (QWER user). Alternatively, <kbd>M</kbd> could be used.

Similarly, there are no hotkeys for the workplane tools.

**Comment by hlorus** [comment]
Hmm not sure i get this right, you are proposing to add another shortcut to switch to the select tool?
Note that the shortcuts in the addon are a bit special as they are defined per workspacetool and therefore only work if one of the addons tools is active. It might be a great addition to add a global shortcut to the select or add sketch tool.

**Comment by Intuos5** [comment]
Yes I am aware, I manually fixed this for myself. I added the Workplane tool to <kbd>Tilda</kbd> and the Select tool to <kbd>W</kbd>. I quite like that it is tool specific so in the Workplane tool, I can switch to the Workplane face tool and vice versa using the same hotkey <kbd>Tilda</kbd>.
I really like that hotkeys for CAD Sketcher are only used within the CAD Sketcher environment. So its hotkeys do not compete with any other hotkeys in Blender's Object mode and it basically becomes a semi-workspace of its own.
